corehq/apps/hqmedia/templates/hqmedia/references.html

```diff
@@ -12,8 +12,7 @@
 {% endblock %}
 
 {% block js %} {{ block.super }}
-  <!-- todo add the media player back in -->
```

Comment:
media player not making it back?

Reply:
heh, I don't even know what this comment meant, but I'm pretty sure it's not going too happen